src/pages/List.js

```diff
@@ -38,19 +69,20 @@ const List = props => {
         // Sort the data
         sort="name"
```

Comment:
What else might you pass here besides `name`? Are the items always in alphabetical order—why?